securedrop/sass/journalist.sass

```diff
@@ -101,3 +101,9 @@ button.small-danger, a.btn.small-danger, .btn.small-danger
 
 .journalist-reply__input
   max-width: 700px
+
+.journalist-username__notes
+  font: 12px Sans-Serif
+  font-weight: normal
+  letter-spacing: 0.05px
```

Comment:
Why is this here? I cannot visually tell much of a difference from this fraction of a pixel increase.

Reply:
.05 looks like an erroneous attempt at entering .5, perhaps? I <3 .5px letterspacing, personally.